mmdeploy/pytorch/functions/adaptive_pool.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+
+import torch.nn.functional as F
+from torch.nn.modules.utils import _pair
+
+from mmdeploy.core import FUNCTION_REWRITER
+from mmdeploy.utils import (Backend, get_backend, get_root_logger,
+                            is_dynamic_shape)
+
+
+@FUNCTION_REWRITER.register_rewriter(
+    func_name='torch.nn.functional.adaptive_avg_pool2d')
+def adaptive_avg_pool2d__default(ctx, input, output_size):
+    """Rewrite `adaptive_avg_pool2d` for default backend."""
+    supported_backends = [Backend.TORCHSCRIPT, Backend.NCNN]
```

Comment:
I recommend to add another rewriter with decorator:

```python
@FUNCTION_REWRITER.register_rewriter(
    func_name='torch.nn.functional.adaptive_avg_pool2d', backend=Backend.TORCHSCRIPT.value)
@FUNCTION_REWRITER.register_rewriter(
    func_name='torch.nn.functional.adaptive_avg_pool2d', backend=Backend.NCNN.value)
```

So we can export model without config.